src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -726,6 +726,12 @@ function MoneyRequestConfirmationList({
         ],
     );
 
+    const confirmButtonRef = useRef<View>(null);
+
+    useEffect(() => {
+        confirmButtonRef.current?.focus();
+    }, []);
+
```

Comment:
The focus should be delayed not to disrupt the animation. Please use `useAutoFocusInput`